core/src/io/anuke/mindustry/input/InputHandler.java

```diff
@@ -809,7 +817,7 @@ public void drawArrow(Block block, int x, int y, int rotation, boolean valid){
     void iterateLine(int startX, int startY, int endX, int endY, Cons<PlaceLine> cons){
         Array<Point2> points;
         boolean diagonal = Core.input.keyDown(Binding.diagonal_placement);
-
+        boolean follower = Core.input.keyDown(Binding.dash);
```

Comment:
Instead of using `dash` (which is a button frequently used for movement), a different keybind (say, U or Z) would be preferable. The exact button doesn't matter too much as long as it's not used.